Task4/Task4_2/RationalNumber.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task4_2
+{
+    internal sealed class RationalNumber : IComparable
+    {
+        public int Numerator { get; }
+
```

Comment:
Why denominator is byte? What about number,  for example, 1/1024?

Reply:
Modified